VitalTrackApp/src/screens/Settings.js

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
-import {View, Text, TouchableOpacity, Switch, StyleSheet} from 'react-native';
+
+
+import { View, Text, TouchableOpacity, Switch, StyleSheet } from 'react-native';
 import SignOutButton from '../components/SignoutButton';
+import HomeScreen from './Home';
+import {useNavigation} from '@react-navigation/native';
 
 const settingsData = [
-  {id: 1, title: 'Profile'},
-  {id: 4, title: 'Help'},
+  { id: 1, title: 'Profile' },
+  { id: 4, title: 'Help' },
 ];
 
 const SettingsScreen = () => {
+  const navigation = useNavigation();
+
   return (
     <View style={styles.container}>
       <Text style={styles.title}>Settings</Text>
 
+      <TouchableOpacity style={styles.backButton}
+        onPress={() => navigation.navigate('Home')}>
+        <Text style={styles.backButtonText}>Back to Home</Text>
```

Comment:
Do we need this? I believe theres a built in back button. In my opinon this just looks weird.

Reply:
If there's a built in back button we dont need it. However, on my end there wasn't one so I was getting stuck on the settings page. Ill double check to see what went wrong

Reply:
checkout update: 

https://github.com/user-attachments/assets/9246b5d6-704a-43a6-94a2-4393417db0a6

Reply:
It looks good!